.github/workflows/tests.yml

```diff
@@ -2,38 +2,49 @@ name: Go
 
 on:
   push:
-    branches: [ "main" ]
+    branches: ["main"]
   pull_request:
-    branches: [ "main" ]
+    branches: ["main"]
 
 jobs:
-
   build:
     strategy:
       matrix:
-        go: ['1.18', '1.19', '1.20']
-        os: ['windows-2019', 'windows-2022', 'ubuntu-20.04', 'ubuntu-22.04', 'macos-11', 'macos-12']
+        go: ["1.18", "1.19", "1.20"]
+        os:
+          [
+            "windows-2019",
+            "windows-2022",
+            "ubuntu-20.04",
+            "ubuntu-22.04",
+            "macos-11",
+            "macos-12",
+            "macos-13",
```

Comment:
[q] is this list for `amd64` arch only? or can we also test on `arm64` for Mac M1 / Linux ARM compatibility?

Reply:
🤔 I hadn't thought about multi-arch testing. Let me look into that.